grin/app/GrinCLI.hs

```diff
@@ -92,6 +92,7 @@ pipelineOpts =
   <|> flg (Eff PrintEffectMap) "pe" "Print effect map"
   <|> flg' Lint 'l' "lint" "Checks the well-formedness of the actual grin code"
   <|> flg' (PrintGrin id) 'p' "print-grin" "Prints the actual grin code"
+  <|> flg (FullPrintGrin id) "full-print-grin" "Prints the actual grin code including external functions"
```

Comment:
You generalized the name in the wrong direction. The print should be the full print as default and the (NEW) simple print the special.

Reply:
Renamed the printing operations.